src/server/ranch/RanchDirector.cpp

```diff
@@ -4096,4 +4108,99 @@ void RanchDirector::HandleChangeSkillCardPreset(
     });
 }
 
+void RanchDirector::HandleUpdateDailyQuest(
+  ClientId clientId,
+  const protocol::AcCmdCRUpdateDailyQuest& command)
+{
+  const auto& clientContext = GetClientContext(clientId);
+  const auto characterRecord = _serverInstance.GetDataDirector().GetCharacter(
+    clientContext.characterUid);
+
+  protocol::AcCmdCRUpdateDailyQuestOK response{};
+  characterRecord.Mutable(
+    [&response](data::Character& character)
+    {
+      character.carrots() += 1000;
+
+      response.newCarrotBalance = character.carrots();
+    });
+  response.quest = {command.quest.questId, command.quest.unk_1, command.quest.unk_2, 1};
+  response.unk_1 = 1;
+  response.unk_2 = 1;
```

Comment:
```suggestion
  response.unk_2 = 1;
  
```
Please add a new line to separate the code a bit more